src/MakePotCommand.php

```diff
@@ -618,6 +618,17 @@ protected function extract_strings() {
 				PhpCodeExtractor::fromDirectory( $this->source, $translations, $options );
 			}
 
+			if ( ! $this->skip_php ) {
+				$options = [
+					// Extract 'Title' and 'Description' headers from pattern files.
+					'wpExtractPatterns' => isset( $this->main_file_data['Theme Name'] ),
+					'include'           => array_merge( $this->include, array( 'patterns' ) ),
```

Comment:
@swissspidy Do you know why this hardcodes `patterns` here? Since these are PHP files they should be included by default. Now you explicitly have to exclude the `patterns` when you for example are using `--include=wp-admin/*`. to only include wp-admin strings.

Reply:
I suppose the idea was to make this more efficient and only scan these files when needed.

But you're right that this is already covered by the call above. I'll create a PHP to combine these.

Reply:
#320